src/fe-common/core/fe-help.c

```diff
@@ -232,7 +232,6 @@ static void show_help(const char *data)
 		if (!header) {
 			printtext(NULL, NULL, MSGLEVEL_CLIENTCRAP,
 				  "Irssi commands:");
-			header = TRUE;
```

Comment:
similarly here, if future code would need to display the header, we get the facts right by storing it